packages/core/src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -223,6 +225,15 @@ function ProductDetails({
                 )
               }
               refNumber={showRefNumber && productId}
+              ratingValue={
+                apiConfig.reviewsAndRatings ? rating.average : undefined
+              }
+              reviewsCount={
+                apiConfig.reviewsAndRatings ? rating.totalCount : undefined
+              }
+              reviewsSectionId="reviews-and-ratings"
+              noReviewsText={noReviewsLabel}
+              reviewsCountText={reviewsCountLabel}
```

Comment:
suggestion: thinking about having a single prop `reviewsAndRatings` that sends an object with the info related to reviews and ratings so we don't have a lot of scattered props that is only about reviews and ratings. What do you think?

Reply:
I like this idea.

Reply:
nice!! I was about to suggest it in the component hahah <3